wiki/en/en-Installation-for-Linux.md

```diff
@@ -15,25 +15,35 @@ Make sure you read the [Getting Started](Getting-Started) page.
 Although we don't provide official packages for every Linux distribution, you might find Jamulus in the package manager of your distro. Have a look at this table:
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/jamulus.svg)](https://repology.org/project/jamulus/versions)
-
+P
```

Comment:
Could you please remove this?

Reply:
Sorry about that, and thank you for spotting this.